### ISSUE TITLE
Samsung TV: Crash/Freeze when trying to change selectors sorted by categories (Trakt List) & order (Added Asc)

### ISSUE THREAD

**Issue by SubEdict** [Stremio/stremio-bugs#648]
**Describe the bug**
From the Discover menu: Trakt [1], If you try to change the selectors to anything else than the default like Personal list: "New release" [2] + Sorted by "Added Asc" [3], Stremio will crash/stop responding.

Sample Screenshot **To Reproduce**:
https://i.imgur.com/Y28st7n.jpg

**Update:** This will also apply to any other categories that you try to change, like from the menu More - Movie: [Selector] change to Series.
So, basically only the Home page and library work.

**Expected behavior**
Trakt should show my list and sort by last added first (instead of the default Top).

 - OS: [Tizen]
 - Samsung TV: UE43TU7100 4K, Q95T 4k.
 - Stremio Version [1.0.1]

**Comment by TRtomasz** [comment]
Closing for no if this is still present please let me know in the commebt, it was most likely fiexed already
